LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/IotHubStationInfo.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoRaWan.NetworkServer
+{
+    using Newtonsoft.Json;
+
+    public sealed record IotHubStationInfo
```

Comment:
If this is going to be mutable, it's best to make it a `class`.

Reply:
My mistake, wanted to make it immutable but made a copy/paste error. Will fix this.

Reply:
Done in 0ac22dfde69a85374c69cd5555aafcb99b04a8f4